src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -473,7 +474,7 @@ private static Document GetRequiredDocument(Project project, DocumentId document
 
             private async Task EnqueueWorkItemAsync(Project project, InvocationReasons invocationReasons)
             {
-                foreach (var documentId in project.DocumentIds)
+                foreach (var documentId in project.DocumentIds.Concat(project.AdditionalDocumentIds).Concat(project.AnalyzerConfigDocumentIds))
```

Comment:
This is the core change

Reply:
💭 It may be better to have three separate copies of the block to avoid some of the allocations